msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -72,7 +72,7 @@ public PublicClientApplication(string clientId, string authority)
 
 #if WINRT
 /// <summary>
-/// 
+/// flag to enable logged in user authentication
```

Comment:
When set to true, the application will try to connect to the corporate network using windows integrated authentication?